app/src/main/res/layout/fragment_registration.xml

```diff
@@ -59,18 +59,29 @@
                   android:layout_width="match_parent"/>
     </com.google.android.material.textfield.TextInputLayout>
 
-    <com.google.android.material.textfield.TextInputLayout
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/Mifos.DesignSystem.Spacing.marginBetweenItemsSmall"
-        android:layout_width="match_parent">
 
-        <com.google.android.material.textfield.TextInputEditText
-            android:hint="@string/phone_number"
-            android:id="@+id/et_phone_number"
-            android:inputType="number"
-            android:layout_height="wrap_content"
-            android:layout_width="match_parent"/>
-    </com.google.android.material.textfield.TextInputLayout>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+            <com.hbb20.CountryCodePicker
+                android:id="@+id/countyCodePicker"
```

Comment:
How do we use the selected country code and phone number when registering? I don't see if you have used this id (`countyCodePicker`) anywhere in the code. Could you please implement that part as well

Reply:
Yes, @jawidMuhammadi i haven't used that in the code as the issue I was solving had mentioned only to add country code picker in the registeration fragment and i assumed it to be just a UI change.

Reply:
Adding only the UI doesn't make any sense and there will be no use for it, so please update the issue as well if you can and implement all the necessary changes in this PR. Thanks

Reply:
yes, i can implement the changes , and push them in the next commit

Reply:
@jawidMuhammadi I have added the backend functionality and now it can check at the validity( if the entered number id full or not) of a number at the time of registeration